lib/Target/X86/X86InstrCompiler.td

```diff
@@ -56,6 +56,19 @@ def : Pat<(X86callseq_start timm:$amt1),
           (ADJCALLSTACKDOWN32 i32imm:$amt1, 0)>, Requires<[NotLP64]>;
 
 
+let usesCustomInserter = 1, Defs = [EFLAGS] in {
+def GC_TRANSITION_RA : I<0, Pseudo, (outs), (ins i8mem:$dst),
```

Comment:
What does the '_RA' mean?  I'm pretty sure you don't mean register allocator but I'm not sure what it does refer to.  (Did I miss a comment somewhere?)